app/views/admin/community_transactions/index.haml

```diff
@@ -9,6 +9,9 @@
 .left-navi-section
   %h2= t("admin.communities.transactions.transactions", community_name: community.name(I18n.locale))
 
+  - if feature_enabled?(:export_as_csv)
+    #export-as-csv.right= link_to(" " + t("admin.communities.transactions.export_all_as_csv"), {per_page: 99999, format: :csv}, {class: icon_class("download")})
```

Comment:
I think this could be changed so that the `per_page` is not mandatory, and giving `nil` would mean that it should return everything.